diff --git a/Makefile.am b/Makefile.am
index ef21831..af04605 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -7,7 +7,7 @@ install-exec-hook:
 	chmod 0600 $(DESTDIR)$(sysconfdir)/key_file
 	chmod 0600 $(DESTDIR)$(sysconfdir)/ipmi_pass
 
-securelibdir = /lib/security
+securelibdir = $(base_libdir)/security
 securelib_LTLIBRARIES = pam_ipmicheck.la pam_ipmisave.la
 
 pam_ipmicheck_la_SOURCES = src/pam_ipmicheck/pam_ipmicheck.c
