diff --git a/user_channel/passwd_mgr.cpp b/user_channel/passwd_mgr.cpp
index 939f48f..52442cc 100644
--- a/user_channel/passwd_mgr.cpp
+++ b/user_channel/passwd_mgr.cpp
@@ -51,11 +51,11 @@ struct MetaPassStruct
 {
     char signature[10];
     unsigned char reseved[2];
-    size_t hashSize;
-    size_t ivSize;
-    size_t dataSize;
-    size_t padSize;
-    size_t macSize;
+    uint32_t hashSize;
+    uint32_t ivSize;
+    uint32_t dataSize;
+    uint32_t padSize;
+    uint32_t macSize;
 };
 
 using namespace phosphor::logging;
