diff --git a/configure.ac b/configure.ac
index dcf5e795..6ab005f5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -211,7 +211,7 @@ AS_IF([test "x$POWER_READING_SENSOR" == "x"],[POWER_READING_SENSOR="/usr/share/i
 AC_DEFINE_UNQUOTED([POWER_READING_SENSOR], ["$POWER_READING_SENSOR"], [Power reading sensor configuration file])
 
 AC_ARG_VAR(HOST_IPMI_LIB_PATH, [The file path to search for libraries.])
-AS_IF([test "x$HOST_IPMI_LIB_PATH" == "x"], [HOST_IPMI_LIB_PATH="/usr/lib/ipmid-providers/"])
+AS_IF([test "x$HOST_IPMI_LIB_PATH" == "x"], [HOST_IPMI_LIB_PATH="$libdir/ipmid-providers/"])
 AC_DEFINE_UNQUOTED([HOST_IPMI_LIB_PATH], ["$HOST_IPMI_LIB_PATH"], [The file path to search for libraries.])
 
 # When a sensor read fails, hwmon will update the OperationalState interface's Functional property.
